scripts/README.md

```diff
@@ -5,9 +5,10 @@ This package contains shell scripts and libraries used for running e2e integrati
 ### run-test.sh
 
 `run-test.sh` can run various integration test suites against the current revision in the invoking directory. 
+`run-cyclonus-tests.sh` Runs cyclonus tests against an existing cluster and validates the output
```

Comment:
Not needed now but for the next PR, we can add a test or another workflow for disabling controller in CP and installing the controller in DP followed by conformance tests.

Reply:
Ack, I'll keep it in my backlog